pkgs/applications/networking/instant-messengers/deltachat-desktop/default.nix

```
@@ -1,7 +1,7 @@
 { lib
 , buildNpmPackage
 , copyDesktopItems
-, electron_28
+, electron_30
```

Nit:
```suggestion
, electron
```

and override the version in `all-packages.nix`.

---

Why?

---

https://github.com/NixOS/nixpkgs/blob/aa14af3e9f30695ea6bf62fb334c5044e477a735/pkgs/by-name/README.md#changing-implicit-attribute-defaults but applies to packages that aren't `by-name` as well.

---

Done. Also I find it less readable and I don't expect many people wanting to patch their electron in comparison.